blog/updating_our_tracking_prevalence_metrics.md

```diff
@@ -0,0 +1,98 @@
+falsely counting extra trackers in a given site.
+
+Thus, having a low threshold makes the metric vulnerable to fluctuations from
+changes in data-volume. The end result is that if this threshold is too low then the
+metric is unstable, fluctuating when no real-world change has occurred. If this
+threshold is too high, then it fails to capture the presence of trackers in particular
+low(er)-volume subpages of a given website (e.g. payment pages).
+
+ 2. The long-tail of low traffic sites skews the results.
+
+    During January our dataset counted 1.3 million distinct sites. However the traffic distribution is very skewed to the top few thousand
+    sites. Looking at the `popularity` metric for sites, which measures the relative amount of traffic
+    compared to the most popular site (Google.com), already the 10th most popular site (Pornhub) has
+    just 6% of Google's traffic. By the 100th place, this ratio is 0.6%, and 1000th place this is 0.08%.
+    By the time we are at the 100,000th entry there are only 430 page loads over a month, and at place
+    1,000,000 just 16. This long tail means that, firstly, the impact of tracker presence on these
+    sites is low - the bottom 50% of the 1.3 million sites we see only corresponds to 1% of total
+    traffic - and secondly, the low data volume increases the noise involved in measuring presence.
+
+Due to these difficulties and the complexities arising when explaining site reach, we
+deceded to redefine this metric - making it simpler and more intuitive to the reader, while
+still capturing the prevaluence of the tracker.
+
+Given that the top 10,000 sites account for 75% of page loads in our data, we decided to measure
+the presence across this fixed set of sites. By fixing the denominator of our formula the output
+is no longer influenced by the number of sites observed, which can vary with data volume. This
+metric is also simpler - a ratio over 10,000 is easier for most people to understand, than over
+1.3 million. For example, we now show a site reach of [13 sites](../trackers/realperson.de.html)
+instead of 0.006% previously.
+
+
+As of now the data is updated to use this new metric, under the `site_reach_top10k` key. A further
+ value, `site_avg_frequency` gives the mean presence across these sites.
+
+**How does this metric compare to the previous one?** We back-calculated the new metric for the last
+5 months, and found this makes the site reach for some of the top trackers even more concerning:
+
+- [Google Analytics](../trackers/google_analytics.html) went from being present on 70% of all websites to 8,871 of the top 10k, and there are only 539 sites in the top 10k which _don't_ use [Google Fonts](../trackers/google_fonts.html) on any page.
+- Of the top 1,000 trackers on WhoTracks.Me, only 47 are on over 1,000 sites, and 331 over 100
+sites. This shows just how dominated the tracking landscape by a few trackers monopolising user data.
```

Comment:
"This shows just how dominated the tracking landscape **is** by a few trackers [...]"?